cwms-servers/CWMS_HOME/config/system_config/cwms_system_logging.properties

```diff
@@ -48,13 +47,19 @@ java.util.logging.ConsoleHandler.formatter = rma.util.logging.CustomFormatter
 rma.util.logging.CustomFormatter.format = %t %L: %m
 rma.util.logging.CustomFormatter.date.format = HH:mm:ss
 
-#Logstash Log Handler Setup
-biz.paluch.logging.gelf.jul.GelfLogHandler.host = http:<host>
-biz.paluch.logging.gelf.jul.GelfLogHandler.port = <port>
-biz.paluch.logging.gelf.jul.GelfLogHandler.level = ALL
+# Enterprise Log Handler Setup
+biz.paluch.logging.gelf.jul.GelfLogHandler.host = <host>
+biz.paluch.logging.gelf.jul.GelfLogHandler.level = FINE
 biz.paluch.logging.gelf.jul.GelfLogHandler.maximumMessageSize = 5000
 biz.paluch.logging.gelf.jul.GelfLogHandler.extractStackTrace = true
 
 
 # sun.rmi.server.call.level=FINER
 # wcds.decode.level=FINER
+
+# Need to keep these packages set to INFO while using e-logging
```

Comment:
whats the deal with this?

Reply:
The SSL layer uses JUL, which routes back to the log handler, which is making HTTPS requests (using SSL), which logs via JUL, and routes back to the log handler...etc This caused a stackoverflow when lowering the log level to FINE.